install-panel.sh

```diff
@@ -45,27 +45,23 @@ if ! [ -x "$(command -v curl)" ]; then
   exit 1
 fi
 
-# define version using information from GitHub
-get_latest_release() {
-  curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
-  grep '"tag_name":' |                                              # Get tag line
-  sed -E 's/.*"([^"]+)".*/\1/'                                      # Pluck JSON value
-}
+#################################
+########## Variables ############
+#################################
 
-echo "* Retrieving release information.."
-PTERODACTYL_VERSION="$(get_latest_release "pterodactyl/panel")"
-echo "* pterodactyl/panel @ $PTERODACTYL_VERSION"
+# versioning
+GITHUB_SOURCE="master"
+SCRIPT_RELEASE="canary"
 
-# variables
-WEBSERVER="nginx"
+# No default FQDN
```

Comment:
Why is it important that the FQDN variable does not have a default value?

Reply:
Not really that important :)